internal/oci/runtime_vm.go

```diff
@@ -749,6 +749,9 @@ func (r *runtimeVM) createContainerIO(ctx context.Context, c *Container, cioOpts
 	defer func() {
 		if retErr != nil {
 			containerIO.Close()
+			if err := os.Remove(c.logPath); err != nil {
+				log.Warnf(ctx, "Failed to remove log path %s after failed to create container: %v", c.logPath, err)
+			}
```

Comment:
Should this be in CreateContainer() rather than createContainerIO() ?

Reply:
hahaha right! fixed :sweat_smile: